### ISSUE TITLE
Dialogs with no focusable elements have an outline around their content

### ISSUE THREAD

**Issue by richardszalay** [likeastore/ngDialog#203]
If there are no focusable elements on the dialog, the first content element is force-focused. This improves compatibility with non-aria-supporting screenreaders, but displays an annoying outline around the text. This could easily be avoided using `.css('outline', '0')`

PR incoming
